cpp/src/reductions/reductions.cpp

```diff
@@ -65,13 +77,30 @@ struct reduce_dispatch_functor {
     case aggregation::VARIANCE: {
       auto var_agg = static_cast<std_var_aggregation const*>(agg.get());
       return reduction::variance(col, output_dtype, var_agg->_ddof, mr, stream);
-      }
-      break;
+      } break;
```

Comment:
What's wrong with giving `break` it's own line? Is this a new clang-format thing?

Reply:
Yes. clang-format puts them in same line!